config/foreman.migrations/20160516143923_remove_ovirt_provision.rb

```diff
@@ -1,3 +0,0 @@
-# Remove unmaintained ovirt_provision plugin
```

Comment:
I think it's safe to delete this migration, but I don't know how kafo handles missing migrations if they would have been applied.